index.js

```diff
@@ -1,29 +1,19 @@
-'use strict';
 require('dotenv').config();
-// require("isomorphic-fetch");
-
-// const ngrokAuthToken = process.env.NGROK_AUTH_TOKEN;
-// const slackOauthToken = process.env.SLACK_OAUTH_TOKEN;
-// const postUrl = process.env.SLACK_POST_URL;
-// const IncomingWebhook = require('@slack/client').IncomingWebhook;
-
-const events = require('./controllers/events.controller');
 
+const Events = require('./controllers/events.controller');
```

Comment:
Usually you reserve the entity name for the model (in singular, capitalized). I use EventsController (in case of a class) or eventsController (instance).